common/on_action/wc_aging_penalties.txt

```diff
@@ -0,0 +1,165 @@
+﻿on_birthday = {
+	on_actions = {
+		on_birthday_aging
+	}
+}
+on_birthday_aging = {
+	trigger = {
+		30 <= { value = age divide = racial_age_multiplier_value }
+	}
+
+	effect = {
+		if = {
+			limit = {
+				has_trait_with_flag = can_not_get_sick
+				NOT = { has_trait = being_light }
+				NOT = { has_trait = being_void }
+			}
+			clear_age_penalties = yes
+		}
+		else = {
+			save_temporary_scope_value_as = {
+				name = my_human_age_multiplier
```

Comment:
Not sure why it's called `my_human_age_multiplier`. It gets the mutiplayer for all the races, not only humans.